### PR TITLE
Stempeg

### PR DESCRIPTION
**Pull request by Faylixe**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/stempeg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "stempeg" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (marcelotrevisani) [inline recipes/stempeg/meta.yaml]:
```suggestion
  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```

File: recipes/stempeg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    sha256: f2266ad32d4e4ff3a936279b62a48ba58e6ada32f0e5ee53f4968877c0148136
```
Comment (marcelotrevisani) [inline recipes/stempeg/meta.yaml]:
```suggestion
  sha256: f2266ad32d4e4ff3a936279b62a48ba58e6ada32f0e5ee53f4968877c0148136
```

File: recipes/stempeg/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - ffmpeg
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/stempeg/meta.yaml]: just to double-check, do you really need ``setuptools`` to run this project?

Comment (marcelotrevisani) [reply recipes/stempeg/meta.yaml]: I think we can remove it, right?
```suggestion
```

Comment (Faylixe) [reply recipes/stempeg/meta.yaml]: Good question, I will check that (I am losing myself in the ocean of dependencies to adapt in Conda package in order to have our library available and kinda get confused sometimes ^^')

Comment (Faylixe) [reply recipes/stempeg/meta.yaml]: Effectivly, `setuptools` is not a direct dependencies, only used by `setup.py` provided script. I remove it :)

Comment (marcelotrevisani) [reply recipes/stempeg/meta.yaml]: It seems it is a direct dependency, somewhere is importing ``pkg_resources`` which is distributed with ``setuptools`` :/

Comment (marcelotrevisani) [reply recipes/stempeg/meta.yaml]: https://github.com/faroit/stempeg/blob/256f460b176d7508b6d71a3607b4435fc3350efc/stempeg/__init__.py#L13

Comment (Faylixe) [reply recipes/stempeg/meta.yaml]: Actually it is needed for `pkg_resources` module so I will restore it

**Comment by marcelotrevisani** [comment]
You will have to skip windows for this recipe, there is no ``libsndfile`` for windows

**Comment by marcelotrevisani** [comment]
I will push one modification

**Comment by Faylixe** [comment]
> You will have to skip windows for this recipe, there is no `libsndfile` for windows

Because it is not available on Conda ? The dependency to `pysoundfile` is described as working under *Windows* platform.

**Comment by marcelotrevisani** [comment]
> > You will have to skip windows for this recipe, there is no `libsndfile` for windows
> 
> Because it is not available on Conda ? The dependency to `pysoundfile` is described as working under _Windows_ platform.

Yeah, but ``pysoundfile`` rely on ``libsndfile``
That recipe is wrong, actually... I opened a PR to change it
However, if you want to give a try, you can try to build ``libsndfile`` for windows and it will solve all those things.

**Comment by Faylixe** [comment]
> > > You will have to skip windows for this recipe, there is no `libsndfile` for windows
> > 
> > 
> > Because it is not available on Conda ? The dependency to `pysoundfile` is described as working under _Windows_ platform.
> 
> Yeah, but `pysoundfile` rely on `libsndfile`
> That recipe is wrong, actually... I opened a PR to change it
> However, if you want to give a try, you can try to build `libsndfile` for windows and it will solve all those things.

I actually think I will for a next release of this package, a first iteration with only UNIX platform support is OK for us by now. I will update parent package `musdb` as well to restrict usage and exclude windows platform then.

Thanks you for the whole review :)

**Comment by Faylixe** [comment]
(is it possible to merge now then so I can move forward on `musdb` package ?)

**Pull request merged**